File: syntax_error.js
// JavaScript with a syntax error
function calculateSum(arr) {
    let total = 0;
    for (let num of arr) // <--- Syntax error: Missing opening brace
        total += num;
    }
    return total;
}

let numbers = [1, 2, 3, 4, 5];
let result = calculateSum(numbers);
console.log("Sum in JavaScript:", result);

File: syntax_error.cpp
// C++ with a syntax error
#include <iostream>
using namespace std;

int calculateSum(int arr[], int size) {
    int total = 0;
    for (int i = 0; i < size; i++) {
        total += arr[i] // <--- Syntax error: Missing semicolon
    }
    return total;
}
// ... main function ...
